python/hail/context.py

```diff
@@ -465,6 +471,74 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
         return VariantDataset(self, jvds)
 
     @handle_py4j
+    def import_vcf_generic(self, path, force=False, force_bgz=False, header_file=None, npartitions=None,
```

Comment:
what about one method with a generic flag? that would fit the sense of the rest of these methods